nrf_dm: Adding version 0.7.0 of nrf_dm (#735)
nrfconnect/sdk-nrfxlib: nrf_dm/doc/nrf_dm_usage.rst

   You can meet this synchronization requirement by starting the reflector earlier and increasing the timeout value, at the cost of higher power consumption.

#. Load the raw data using the :c:func:`nrf_dm_populate_report` function.

#. Call the :c:func:`nrf_dm_calc` function or the :c:func:`nrf_dm_high_precision_calc` function to perform the calculation.

>    On the nRF53 Series, this step should preferably be done on the application core for maximum performance.

Reviewer: ```suggestion
   On the nRF53 Series, do this on the application core for maximum performance.
```

>    On the nRF53 Series, this step should preferably be done on the application core for maximum performance.

Reviewer 2: Maybe we should specify that performance you are talking about is execution time (and not anything to do with the accuracy of the estimates)
